### ISSUE TITLE
Documentation/tutorial on using gl4es with GLU on Emscripten

### ISSUE THREAD

**Issue by Sgeo** [ptitSeb/gl4es#352]
Maybe there's some basic C/C++ I don't understand, but I'm having some difficulty using GLU with gl4es on Emscripten. Tried a bunch of things, and I keep getting linker errors. Sometimes stuff like `error: undefined symbol: mgluScaleImage (referenced by top-level compiled C/C++ code)`, sometimes stuff like `glEnd` being missing.

A tutorial or documentation on using these together would be a nice thing to have, in my opinion.

I do note that gl4es defines `USE_MGL_NAMESPACE` for Emscripten and Apple, and GLU only does so for Apple. The only way I could think of to fix that is to tamper with the header in GLU.

**Comment by ptitSeb** [comment]
Yeah, that could work. GL needs to be mangled in Emscripten, but not GLU.

**Comment by major-mayer** [comment]
Hi @Sgeo @ptitSeb 
Could you maybe explain a bit more in detail what you did to get GLU running when compiling with Emscripten?
Glu is shipped with Emscripten, so i don't think that i need to recompile it myself right?

If i am not changing anything inside the source code of gl4es i get the following error:
```
[build] wasm-ld: error: ...example.a(example.cpp.o): undefined symbol: mgluUnProject
```

I've tried to comment out `#include "glu_mangle.h"` inside glu.h but this time i get a different result:
```
[build] wasm-ld: error: ...example.a(example.cpp.o): undefined symbol: gluUnProject
```
So it seems like i am missing an exported version of the unmangled names right?
What do you mean when you say "I'm trying for now is copying the gl4es headers to GLU". Did you copy glu.h to the Emscripten GLU include folder?
